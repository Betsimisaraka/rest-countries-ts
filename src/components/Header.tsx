import React from 'react';

function Header() {
    return (
        <div>
            <h1 data-testid="page-title">Where in the world</h1>
            <p>Dark mode</p>
        </div>
    )
}

export default Header
