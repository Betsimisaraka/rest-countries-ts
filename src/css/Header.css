.header {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    background-color: hsl(0, 0%, 100%);
    box-shadow: 2px 2px 2px hsl(0, 0%, 52%);
    padding: 22px 22px;
}

.dark {
    background-color: hsl(209, 23%, 22%);
    color: white;
}

h1 {
    font-size: 1.2rem;
    font-weight: 700;
}

p {
    font-size: 14px;
    font-weight: 700;
}

h1, p {
    margin: 0;
    padding: 0;
}