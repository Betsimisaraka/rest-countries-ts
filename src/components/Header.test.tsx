import React from 'react';

import { render } from '@testing-library/react';
import Header from './Header';

test('<Header /> ', () => {
    const { queryByTestId, container } = render(<Header />);
    const title = queryByTestId("page-title");
    expect(title).toBeTruthy(); 
    // expect(container.firstChild).toMatchSnapshot();
})
