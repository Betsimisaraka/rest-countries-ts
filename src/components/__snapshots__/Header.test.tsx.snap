// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header />  1`] = `
<div>
  <h1
    data-testid="page-title"
  >
    Where in the world
  </h1>
  <p>
    Dark mode
  </p>
</div>
`;
